singolo/js/script.js

```diff
@@ -0,0 +1,164 @@
+// This Array with images for portfolio
+let imgArray = [
+    './src/img/portfolio-1.png',
+    './src/img/portfolio-2.png',
+    './src/img/portfolio-3.png',
+    './src/img/portfolio-4.png',
+    './src/img/portfolio-5.png',
+    './src/img/portfolio-6.png',
+    './src/img/portfolio-7.png',
+    './src/img/portfolio-8.png',
+    './src/img/portfolio-9.png',
+    './src/img/portfolio-10.png',
+    './src/img/portfolio-11.png',
+    './src/img/portfolio-12.png'
+]
+
+// This header nav menu
+const menu = document.querySelector('.menu');
+// This nav menu elements li
+const menuItems = menu.querySelectorAll('li');
+
+// This tags menu
+const tags = document.querySelector('.tags');
+// This tags menu elements li
+const tagsItems = tags.querySelectorAll('li');
+
+// This portfolio container for images
+const portfolio = document.querySelector('.portfolio-images');
```

Comment:
I have not found the usage of this variable, so this and the next line can be united:

```
const portfolioImages = portfolio = document.querySelector('.portfolio-images').querySelectorAll('img');
```

One more comment according to the naming. I know, sometimes choosing the name can took much more time than implementing the functionality) But you should always try to use such the names, giving the other team members the understanding what is the variable responsible for without getting dipper in the code)